src/store/actions.ts

```diff
@@ -82,6 +82,16 @@ export interface SetSearchParams {
   payload: SearchParams;
 }
 
+export interface SetSelectedFacilityId {
+  type: 'SET_SELECTED_FACILITY_ID';
+  payload: string;
+}
+
+export interface ResetSelectedFacilityId {
+  type: 'RESET_SELECTED_FACILITY_ID';
+  payload: undefined;
```

Comment:
you can remove payload from the interface definition because payload is udefined.